netbox/templates/inc/nav_menu.html

```diff
@@ -183,6 +183,26 @@
                         <li{% if not perms.dcim.view_interface %} class="disabled"{% endif %}>
                             <a href="{% url 'dcim:interface_connections_list' %}">Interface Connections</a>
                         </li>
+                        <li class="divider"></li>
+                        <li class="dropdown-header">Device Components</li>
```

Comment:
Console ports and console server ports are missing from the navigation menu.

Reply:
Will fix this up right away